calico-enterprise_versioned_docs/version-3.14/visibility/elastic/troubleshoot.mdx

```diff
@@ -122,7 +122,7 @@ As a last resort, create a new [Elasticsearch cluster](#how-to-create-a-new-clus
 
 **Solution/workaround**: Verify that the intrusion detection job is running, or try removing and reapplying:
 
-```
+```bash
```

Comment:
Have you checked the output on blocks like this with the $ prompt? Do we want to remove?

Reply:
If we're happy to remove, a global s/$ kubectl/kubectl would probably do the trick.

Reply:
Yes, let's remove it. The $ symbol is redundant.

Reply:
Fixed